pages/avoapi/authentication.mdx

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
I'd like to suggest we rename the parent directory from `avoapi` to just `api` to keep the urls nice and clean

Reply:
ah turns out "api" is reserved for endpoints, going for public-api instead in Updated in cf0f9079cfa32e8fd33afd4b74e36ce6e38306bd